.env

```diff
@@ -1,3 +1,5 @@
 REACT_APP_CHAIN_ID="43114"
 REACT_APP_NETWORK_URL="https://node.snowapi.net/ext/bc/C/rpc"
 REACT_APP_ANALYTICS_API="https://dev-axial-api.snowapi.net"
+REACT_APP_ANALYTICS_ENDPOINT='//analytics.snowapi.net/'
```

Comment:
today ill be deploying the new matomo instance with a different url, so I think is best if we wait a bit longer so we can inlcude it here.

Reply:
ok